recipes/GaitPy/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "gaitpy" %}
+{% set version = "1.6.0" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/matt002/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 3b0001220a2f1f19df638b3e26f9ab6838104decbf2dabf303aa71d6cdd29c42
+
+build:
+  skip: True  # [py!=36]
```

Comment:
Since this package doesn't depend on the system architecture, please remove the `skip` line and use `noarch: python` instead. Then, in the `host` and `run` sections below you can specify the python versions you need, e.g. `python >=3.6`.